tests/Controller/JsonApiTestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Sylius\Tests\Controller;
+
+class JsonApiTestCase extends \Lakion\ApiTestCase\JsonApiTestCase
```

Comment:
Actually, do we really need inheritance here? Can't we create an interface with constants and use them in tests?